.github/workflows/Django-CI.yml

```
@@ -2,17 +2,17 @@ name: Django CI
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "Django", "Spring"]
```

대괄호 붙어있는거 너무 불편한데 수정 부탁드립니다..!
```suggestion
    branches: [ "main", "Django", "Spring" ]
```

---

네!